### PR TITLE
.ci/aws: Make failures happen in correct stage

### PR DESCRIPTION
**Pull request by a-szegel**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by a-szegel** [comment]
bot:aws:retest

**Comment by a-szegel** [comment]
bot:aws:retest

File: .ci/aws/Jenkinsfile [hunk 1]
```diff
@@ -84,16 +81,12 @@ def run_test_orchestrator_once(run_name, build_tag, os, instance_type, instance_
...
-                  )
-    if (ret == 65)
-        unstable('Scripts exited with status 65')
-    else if (ret != 0)
-        build_ok = false
-    catchError(buildResult: 'SUCCESS', stageResult: 'FAILURE') {
-        sh "exit ${ret}"
+
+    try {
+        sh ". venv/bin/activate; ./PortaFiducia/tests/test_orchestrator.py ${args}"
```
Comment (rauteric) [inline .ci/aws/Jenkinsfile:86]: Why don't you have to check the return value of this shell call?

Comment (a-szegel) [reply .ci/aws/Jenkinsfile:86]: it throws when there is a non-0 exit code.

Comment (a-szegel) [reply .ci/aws/Jenkinsfile:86]: maybe we don't need the try/catch though

**Review by rauteric — APPROVED** [review]

**Comment by a-szegel** [comment]
bot:aws:retest infra issue, trying again

**Pull request merged**